narwhals/_dask/namespace.py

```diff
@@ -135,6 +137,52 @@ def any_horizontal(self, *exprs: IntoDaskExpr) -> DaskExpr:
     def sum_horizontal(self, *exprs: IntoDaskExpr) -> DaskExpr:
         return reduce(lambda x, y: x + y, parse_into_exprs(*exprs, namespace=self))
 
+    def concat(
+        self,
+        items: Iterable[DaskLazyFrame],
+        *,
+        how: str = "vertical",
+    ) -> DaskLazyFrame:
+        import dask.dataframe as dd  # ignore-banned-import
+
+        if len(list(items)) == 0:
+            msg = "No items to concatenate"
+            raise ValueError(msg)
+        native_frames = [i._native_frame for i in items]
+        axis: int
+        join: str
+        if how == "vertical":
+            if not all(
+                tuple(i.columns) == tuple(native_frames[0].columns) for i in native_frames
+            ):
```

Comment:
I agree that we are introducing this check just now and for other dataframe can follow later on another PR.

As no test is introduced to check for this case, you can add a `# pragma: no cover` and come back to this later on, otherwise CI will complain regarding coverage not being 100%

Reply:
I never new what `# pragma: no cover` was until this point, and now I do 🙌